src/Agent/NewRelic/Agent/Extensions/Providers/Wrapper/AspNetCore6Plus/WrapPipelineMiddleware.cs

```diff
@@ -39,7 +39,7 @@ public async Task Invoke(HttpContext context)
             {
                 _agent.Logger.Log(Agent.Extensions.Logging.Level.Finest, "Not instrumenting incoming OPTIONS request.");
 
-                await _next(context);
+                await _next(context).ConfigureAwait(false);
```

Comment:
Similar to the WrapPipelineMiddleware for older aspnetcore versions, I don't know if this could cause problems if someone added a synchronization context.